cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -15,23 +15,28 @@ OPERATOR_GIT_HASH=8d3c30de8ec5a9a0c9eeb84ea0aa16ba2395cd68  # release-4.4
 # This function gets a command string and invoke it
 # until the command returns an empty string or until timeout
 function retry {
+  set +x
```

Comment:
i think we should know what happens,
i see here and there problems and the prints helps

i also feel it doesnt fit in this too generic PR
talked on the other changes in this function

Reply:
I am not sure what problem did you saw could you elaborate?
'rerty' is extensively used and add a lot of noise to the log due to the '-x' flag

Reply:
talking generally, any problem that occur, i find it better that i see the whole flow expanded,
especially while we are working intense on fixing the lane.

Reply:
It doesnt give much, the values that you pass to `retry` dosent change and you can see them, 
the interesting part is the "action" we pass to `retry`, I can print it

Reply:
all the info should be printed please (i mean not just the action, but the arguments such as retires number and so on, at least once)